debezium-core/src/main/java/io/debezium/relational/TableId.java

```diff
@@ -37,6 +37,12 @@ public static TableId parse(String str) {
     public static TableId parse(String str, boolean useCatalogBeforeSchema) {
         String[] parts = str.split("[\\" + '.' + "]");
         if ( parts.length < 0 ) return null;
+        for (int i = 0; i < parts.length; i++) {
+            String part = parts[i];
+            if ( part.startsWith("\"") && part.endsWith("\"") ) {
+                parts[i] = part.substring(1, part.length()-1);
+            }
+        }
```

Comment:
Just FYI - part of my reasoning is that if we ever handle MS SQL then it uses `[` and `]` chars respectively.